src/utils/ts/makeToast.ts

```diff
@@ -0,0 +1,18 @@
+import { toast } from 'react-toastify';
+
+const TOAST_TYPE = {
+  default: 'DEFAULT',
+  success: 'SUCCESS',
+  info: 'INFO',
+  error: 'ERROR',
+  warning: 'WARNING',
+} as const;
+
+type ToastType = keyof typeof TOAST_TYPE;
+
+const makeToast = (type: ToastType, message: string) => toast(message, {
+  type: toast.TYPE[TOAST_TYPE[type]],
+  position: 'top-center',
+});
```

Comment:
openToast 란 네이밍이 의도에 더 맞지 않나 싶어요

```suggestion
const openToast = (type: ToastType, message: string) => toast(message, {
  type: toast.TYPE[TOAST_TYPE[type]],
  position: 'top-center',
});
```

Reply:
`show, open, make` 중 고민을 해봤었는데, **토스트 알림을 생성한다**라는 느낌이라 make로 이름지었습니다.
여러개의 토스트가 겹쳐서 나올 수 있다는 점으로 미루어보면 makeToast도 꽤 적절해보이지 않나요..? ㅎㅎ

Reply:
아하 저는 토스트는 만들어져있고, 정해진 시간만큼 보여졌다가 사라진다고 느껴서요 ㅎㅎ
말씀하신 의도도 좋을듯합니다~